[gazebo] compile error DSO missing from command line

While rebuilding gazebo classic packages I get the following error:
```bash
/sbin/ld: CMakeFiles/ign_msgs_gen.dir/Generator.cc.o: undefined reference to symbol '_ZN4absl12lts_2023012513hash_internal15MixingHashState28CombineLargeContiguousImpl64EmPKhm'
/sbin/ld: /usr/lib/libabsl_hash.so.2301.0.0: error adding symbols: DSO missing from command line
```
This error occurs while building _ignition-mgs-5_ and _gazebo_

--- a/gazebo/.SRCINFO
+++ b/gazebo/.SRCINFO
@@ -1,7 +1,7 @@
 pkgbase = gazebo
 	pkgdesc = A multi-robot simulator for outdoor environments
 	pkgver = 11.13.0
-	pkgrel = 1
+	pkgrel = 2
 	url = https://classic.gazebosim.org/
 	install = gazebo.install
 	arch = i686

--- a/gazebo/.SRCINFO
+++ b/gazebo/.SRCINFO
@@ -31,6 +31,7 @@ pkgbase = gazebo
 	depends = ignition-msgs-5
 	depends = tinyxml2
 	depends = qwt
+	depends = cppzmq
 	optdepends = bullet: Bullet support
 	optdepends = cegui: Design custom graphical interfaces
 	optdepends = ffmpeg4.4: Playback movies on textured surfaces

--- a/gazebo/PKGBUILD
+++ b/gazebo/PKGBUILD
@@ -17,7 +17,7 @@ url="https://classic.gazebosim.org/"
 license=('Apache')
 depends=('boost' 'curl' 'freeglut' 'freeimage' 'tbb' 'libccd' 'libltdl' 'graphviz'
          'libtar' 'libxml2' 'ogre-1.9' 'protobuf' 'sdformat-9' 'ignition-math-6' 'ignition-transport-8'
-         'ignition-cmake-2' 'ignition-common-3' 'ignition-fuel_tools-4' 'ignition-msgs-5' 'tinyxml2' 'qwt')
+         'ignition-cmake-2' 'ignition-common-3' 'ignition-fuel_tools-4' 'ignition-msgs-5' 'tinyxml2' 'qwt' 'cppzmq')
 optdepends=('bullet: Bullet support'
             'cegui: Design custom graphical interfaces'
             'ffmpeg4.4: Playback movies on textured surfaces'
@@ -40,6 +40,7 @@ build() {
   mkdir -p build && cd build
 
   export PKG_CONFIG_PATH=/usr/lib/ffmpeg4.4/pkgconfig
+  export LDFLAGS="-Wl,--copy-dt-needed-entries"
   cmake .. -DCMAKE_BUILD_TYPE="Release" \
            -DCMAKE_INSTALL_PREFIX="/usr" \
            -DCMAKE_INSTALL_LIBDIR="lib"

--- a/gazebo/PKGBUILD
+++ b/gazebo/PKGBUILD
@@ -10,7 +10,7 @@
 
 pkgname=gazebo
 pkgver=11.13.0
-pkgrel=1
+pkgrel=2
 pkgdesc="A multi-robot simulator for outdoor environments"
 arch=('i686' 'x86_64')
 url="https://classic.gazebosim.org/"

